src/Image.cc

```diff
@@ -189,6 +190,25 @@ NAN_METHOD(Image::GetSource){
   info.GetReturnValue().Set(Nan::New<String>(img->filename ? img->filename : "").ToLocalChecked());
 }
 
+/*
+ * Get raw data.
+ */
+NAN_GETTER(Image::GetRawData) {
+  Image *img = Nan::ObjectWrap::Unwrap<Image>(info.This());
+  if (img->_surface) {
+    // Return raw ARGB data -- just a memcpy()
+    cairo_surface_t *surface = img->_surface;
+    cairo_surface_flush(surface);
+    const unsigned char *data = cairo_image_surface_get_data(surface);
+     unsigned int nBytes = img->width * img->height * 4;
```

Comment:
`nBytes = cairo_image_surface_get_stride(surface) * img->height` technically. I don't think images can decode into another surface type now, but this'll be one less thing to change if they can in the future.

Reply:
Great, thx for the feedback. I've changed the code and merged with master.